.circleci/rc.yaml

```diff
@@ -40,4 +41,13 @@ plugins:
     - space 
 # Tables
   - remark-lint-table-pipes
-  - remark-lint-no-literal-urls
```

Comment:
Could you give some details on these changes? The remark rules on all STAC repos are aligned and to give predicatable behavior I'd suggest to keep them as before...

Reply:
This one was apparently out of sync with the one in stac-spec, which is why it was giving so many errors. I updated it with that one and the rules still work fine.

Reply:
Ah, maybe we did mess up with dev/master here? Not sure, I thought I had updated all repos already.